qubes/ext/windows.py

```diff
@@ -49,6 +49,16 @@ def qubes_features_request(self, vm, event, untrusted_features):
             vm.features['os'] = guest_os
         if guest_os == 'Windows' and qrexec:
             vm.features['rpc-clipboard'] = True
+            setattr(vm, 'maxmem', 0)
+            setattr(vm, 'qrexec_timeout', 6000)
```

Comment:
I realize 1min may be too little, but over an hour isn't excessive?

Reply:
Booting after system updates can be time consuming. 

qvm-create-windows-qube default value is 999999
https://github.com/elliotkillick/qvm-create-windows-qube/blob/master/qvm-create-windows-qube#L284

Reply:
In that case we should fix QubesOS/qubes-issues#7116.

Reply:
perhaps it makes sense to disable this timeout (value 0, for example), since even 6000 does not guarantee anything and at the same time does not solve the problems of a hung VM?

Reply:
> In that case we should fix https://github.com/QubesOS/qubes-issues/issues/7116.

Since the above is fixed, I think we can go ahead with merging it.
Later we can adjust to other versions, like no timeout at all. But for that, I'd prefer  better handling for it in various tools (for example `qvm-start` would basically hang).